chanzuckerberg/single-cell-data-portal
# feat: Add survey bottom banner without hubspot

## Reason for Change

- #7313 

## Changes

- add BottomBanner with new Airtable Survey link to Cell Guide, Differential Expression, & Gene Expression
- remove references to Hubspot  - their cookie policy does not work with our privacy policy
- remove Newsletter sign up on landing page 
- modify BottomBanner component to conditionally allow Newsletter Signup in case we need to activate that functionality in the future - no Banners are currently using that functionality though.

### Bottom Banner Visible and Clickable
<img width="1436" alt="Screenshot 2024-07-31 at 11 52 56 AM" src="https://github.com/user-attachments/assets/5a7104cf-67e8-4e06-9194-c616c3a6f2e2">

### No Newsletter signup on Landing page Footer
<img width="1422" alt="Screenshot 2024-07-31 at 11 54 06 AM" src="https://github.com/user-attachments/assets/0ec71ed4-236b-427a-a280-b45dac32f2dc">

## Testing steps

- Check Cell Guide, Differential Expression, & Gene Expression and see bottom banner 
- Click survey link to take you to correct airtable survey
- Click x button to dismiss banner - it will not return for 30 days after you do this (except in incognito browser)
- Check bottom banner on mobile screens as will as large. 

## Checklist 🛎️
